aspnet/web-pages/overview/security/16-adding-security-and-membership.md

```diff
@@ -204,7 +204,7 @@ The login page will not stop automated programs (sometimes referred to as *web r
 
 ![/media/38777/ch16securitymembership-18.jpg](16-adding-security-and-membership/_static/image1.jpg)
 
-1. Register your website at ReCaptcha.Net ([http://recaptcha.net](http://recaptcha.net)). When you've completed registration, you'll get a public key and a private key.
+1. Register your website the [Recaptcha service](https://www.google.com/recaptcha/about/). When you've completed registration, you'll get a public key and a private key.
```

Comment:
I suggest a slightly different link. This one is referenced on https://developers.google.com/recaptcha/intro, where it specifically talks about signing up for the API key pair:
```suggestion
1. Register your website with the [ReCaptcha service](http://www.google.com/recaptcha/admin). When you've completed registration, you'll get a public key and a private key.
```
(see my note below)